### PR TITLE
add JO password: UI, Connect

### PR DESCRIPTION
**Pull request by Yulllia**
image
-added ability change password by user with old password

### CODE REVIEW

**Comment by lyudad** [comment]
show password better to put under input, try to use same approach as other apps use

File: src/pages/setting-page-client/change-password/ChangePassword.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	email: string;
+};
+
+export interface Error {
+	data: {
+		message: string;
+	};
+}
+type Alert = 'success' | 'error';
+const schema = Yup.object({
```
Comment (lyudad) [inline src/pages/setting-page-client/change-password/ChangePassword.tsx]: better move to separate file

File: src/pages/setting-page-client/change-password/ChangePassword.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+			try {
+				await setPassword({
+					oldPassword: oldPassword,
+					newPassword: newPassword,
+					email: user.email || '',
+				}).unwrap();
+				reset({ oldPassword: '', newPassword: '', confirmPassword: '' });
+				alert('success', 'Password changed successfully!');
+			} catch (e) {
+				reset({ oldPassword: '', newPassword: '', confirmPassword: '' });
```
Comment (lyudad) [inline src/pages/setting-page-client/change-password/ChangePassword.tsx]: you duplicate { oldPassword: '', newPassword: '', confirmPassword: '' } very oten, move to constant please

File: src/pages/setting-page-client/change-password/ChangePassword.tsx (hunk 1 above)
Comment (Yulllia) [reply src/pages/setting-page-client/change-password/ChangePassword.tsx]: fixed

File: src/pages/setting-page-client/change-password/ChangePassword.tsx (hunk 2 above)
Comment (Yulllia) [reply src/pages/setting-page-client/change-password/ChangePassword.tsx]: fixed

**Pull request merged**